examples/mcmc.jl

```diff
@@ -137,26 +127,27 @@ function data_generating_procedure(t::Int64)
         _choices = Gen.get_submap(choices, addr)
         Gen.set_submap!(obs, addr, _choices)
     end
-
-    #=  plt = lineplot(1:t, get_zs(trace),
-                   title="Height of ball", name="observation with res $(trace[:prior => 1 => :restitution])",
-                   xlabel="t", ylabel="z", canvas=DotCanvas,
-                   border=:ascii) =#
-    plt = plot(1:t, get_zs(trace),title="Height of ball", xlabel="t", ylabel="z",label="Observation")
     
-    return (gargs, obs, plt)
+    return (gargs, obs)
```

Comment:
you could return the ground truth trace here if you wanted to access to the original set of latents. this will be good to include in your inference plots.